lib/kvs/kvs_rparams.c

```diff
@@ -786,6 +786,21 @@ static const struct param_spec pspecs[] = {
             },
         },
     },
+    {
+        .ps_name = "cn_incr_rspill",
```

Comment:
Will this parameter go away when this work is done?

Reply:
Yes, it will. This is a temporary knob to let us go back to the old behavior.